modules/concurrency/include/hephaestus/concurrency/spinner.h

```diff
@@ -29,6 +30,7 @@ class Spinner {
 
   void start();
   auto stop() -> std::future<void>;
+  void waitForCompletion();
```

Comment:
nit. this should be called `wait` as we use `wait` elsewhere

Reply:
Renamed.